src/Integration/Flysystem/S3/composer.json

```diff
@@ -27,7 +27,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.3-dev"
+            "dev-master": "0.4-dev"
```

Comment:
Every package have a different version. That's makes the things hard to maintain. 

What's about using the same version everywhere? Even when nothing change between 2 versions?

Reply:
We only have this issue now when we are breaking BC all the time. 

After version 1 it will be smooth sailing